plugins/doc_fragments/foreman.py

```diff
@@ -45,3 +45,38 @@ class ModuleDocFragment(object):
     default: true
     type: bool
 '''
+
+    NESTED_PARAMETERS = '''
+options:
+  parameters:
+    description:
```

Comment:
I think if you have `description` set here *and* in the individual modules, you end up with both descriptions joined. Which is not what we want.

Reply:
I tested, and at least ansible-doc only takes the one from the module. So i let it be a fallback description.

Reply:
ok, then my memory is playing games with me. thanks for testing.